docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,143 @@
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# ParaTime Observer Node
+
+:::info
+
+These instructions are for setting up a _ParaTime observer_ node, which observes ParaTime activity and can issue
+queries that can access confidential data. If you just want to run a _ParaTime client_ node, see the
+[instructions for running a ParaTime client node](paratime-client-node.mdx). If you want to run a _ParaTime_ node
+instead, see the [instructions for running a ParaTime node](paratime-node.mdx). Similarly, if you want to run a
+_validator_ or a _non-validator_ node instead, see the [instructions for running a validator node](validator-node.mdx)
+or [instructions for running a non-validator node](non-validator-node.mdx).
+
+:::
+
+:::tip
+
+The observer node role allows any ParaTime client node with appropriate TEE support to run confidential transactions
```

Comment:
This is reads incorrectly as if any client node with TEE (e.g. Emerald client node with TEE), can run confidential queries in Sapphire.

It is also missing the information that some ParaTimes require that the entity used by observer nodes is whitelisted.

From the requirements perspective the most important things that they need a machine with TEE support and register an entity.

I believe this alert box could be merged into the first paragraph in a concise manner pointing out only what is important.

Reply:
I have rephrased this and merged it into the paragraph below.